src/circuit/circuit_frame.rs

```diff
@@ -2833,6 +2866,10 @@ fn apply_continuation<F: LurkField, CS: ConstraintSystem<F>>(
                     key: Op2::Cons.as_field(),
                     value: cons.hash(),
                 },
+                CaseClause {
+                    key: Op2::StrCons.as_field(),
+                    value: cons.hash(),
```

Comment:
Shouldn't this be `strcons.hash()` (assuming the existence of a `strcons` binding)?

Reply:
Or, maybe this is taking advantage of the fact that the hash will be the same in either case. If that's what's going on, I think at least some comments will help clarify.

If I'm not mistaken, that means the `construct_strcons` function is not actually used anywhere.

Reply:
> Shouldn't this be `strcons.hash()` (assuming the existence of a `strcons` binding)?

The hash is the same, only the tag differs.